community/cypher/cypher/src/test/scala/org/neo4j/cypher/internal/RewindableExecutionResult.scala

```diff
@@ -289,9 +289,10 @@ object RewindableExecutionResult {
     override def javaColumnAs[T](column: String): ResourceIterator[T] = inner.javaColumnAs(column)
 
     override def executionPlanDescription(): InternalPlanDescription = {
-      var description = lift(inner.executionPlanDescription())
-      if (!description.arguments.exists(_.isInstanceOf[Arguments.Version])) {
-        description = description.addArgument(planDescription.InternalPlanDescription.Arguments.Version("CYPHER 3.1"))
+      var description = lift(inner.executionPlanDescription()).addArgument(Arguments.PlannerVersion("3.1"))
+      if (!description.arguments.exists(_.isInstanceOf[Arguments.RuntimeVersion])) {
+        // FIXME do we need this
+        description = description.addArgument(Arguments.RuntimeVersion("CYPHER 3.1"))
```

Comment:
I wonder if this works around bugs in some older version of the compiler for 3.1, and if that bug is fixed, then this workaround could be removed. Perhaps it is as simple as removing the workaround and running all compatibility tests?

Reply:
Since the "remove duplicate tx wrappers" PR changes that code already (and removes the unnecessary parts), I am not changing it here, to avoid conflicts.